src/OSPSuite.UI/Views/Journal/JournalPageEditorView.cs

```diff
@@ -220,6 +225,15 @@ public override void InitializeResources()
          layoutItemSource.Text = Captions.Journal.Source.FormatForLabel();
 
          cbSource.SetImages(_imageListRetriever);
+
+         ribbonControl.ShowToolbarCustomizeItem = false;
```

Comment:
I found some bugs with the ApplicationButton and the toolbar today. So I decided to simply hide them (we do not use them anyways)